src/features/Instructor/Profile/index.jsx

```diff
@@ -93,19 +87,14 @@ const Profile = () => {
             {classes.data.length > 0 && (
               <>
                 <h3 className="text-uppercase">Courses taught</h3>
-                {classes.data?.slice(0, CLASS_LIMIT).map(({ classId, className }) => {
-                  const url = addQueryParam(`/classes/${classId}?previous=my-profile`);
-
-                  return (
-                    <Link
-                      to={url}
-                      key={classId}
-                      className="text-truncate link"
-                    >
-                      {className}
-                    </Link>
-                  );
-                })}
+                {classes.data?.slice(0, CLASS_LIMIT).map(({ classId, masterCourseName }) => (
```

Comment:
There won't be a limit of the courses list, please remove it

Reply:
Done